doc/api/util.md

```diff
@@ -1176,7 +1697,8 @@ added: v0.11.5
 deprecated: v4.0.0
 -->
 
-> Stability: 0 - Deprecated
+> Stability: 0 - Deprecated: Use
+> `typeof value === 'object' && typeof value !== 'function' ` instead.
```

Comment:
This function is implemented as `value !== null && typeof value === 'object'`.

Maybe also mention that even though [functions are also objects](https://timothygu.me/es-howto/#function-object), `isObject` historically returned `false` even for functions, because it's broken.

The doc as-is doesn't make much sense.